Lib/Database/MySqlQuery.php

```diff
@@ -0,0 +1,1247 @@
+<?php
+/**
+ * MySqlQuery class file.
+ *
+ * @package Freyja\Database
+ * @copyright 2016 SqueezyWeb
+ * @author Gianluca Merlo <gianluca@squeezyweb.com>
+ * @since 0.1.0
+ */
+
+namespace Freyja\Database;
+
+use Freyja\Exceptions\InvalidArgumentException as InvArgExcp;
+use RuntimeException;
```

Comment:
We could create a `Freyja\Exceptions\RuntimeException` too. Then, with a `Freyja\Exceptions\ExceptionInterface` we can target all exceptions thrown by this project.

See [SqueezyWeb/Exceptions#1](https://github.com/SqueezyWeb/Exceptions/issues/1)